src/main/java/org/sagebionetworks/web/client/jsinterop/SRC.java

```diff
@@ -18,6 +20,8 @@
 		public static ReactFunctionComponent<SchemaDrivenAnnotationEditorProps> SchemaDrivenAnnotationEditor;
 		public static ReactFunctionComponent<SynapseNavDrawerProps> SynapseNavDrawer;
 		public static ReactFunctionComponent<EmptyProps> FavoritesPage;
+		public static ReactFunctionComponent SynapseToastContainer;
+		public static native void displayToast(String variant, String title, String description, Long autoCloseInMs, String primaryButtonText, Callback onPrimaryButtonClick, String secondaryButtonText, String secondaryButtonHref);
```

Comment:
I'm not sure if I am ok with exporting a function `displayToast` from `SynapseComponents`, because it's not a component. But I also don't want to export a whole new object in SRC for one function

Reply:
The signature should be pretty stable for this particular case (long history of showing these notifications), but I'm also weary in recommending this.